shared/contracts/BaseTemplate.sol

```diff
@@ -39,10 +39,10 @@ contract BaseTemplate is APMNamehash, IsContract {
     string constant private ERROR_MINIME_FACTORY_NOT_PROVIDED = "TEMPLATE_MINIME_FAC_NOT_PROVIDED";
     string constant private ERROR_MINIME_FACTORY_NOT_CONTRACT = "TEMPLATE_MINIME_FAC_NOT_CONTRACT";
 
-    ENS public ens;
-    DAOFactory public daoFactory;
-    MiniMeTokenFactory public miniMeFactory;
-    IFIFSResolvingRegistrar public aragonID;
+    ENS internal ens;
+    DAOFactory internal daoFactory;
+    MiniMeTokenFactory internal miniMeFactory;
+    IFIFSResolvingRegistrar internal aragonID;
```

Comment:
Hmm, I do like having these stay `public`; I do use them quite a bit when debugging / figuring out what a template is attached to (although I guess I could look up the contract creation transaction, but that's a bit more involved).